cypress/e2e/bookStore.cy.js

```diff
@@ -2,10 +2,74 @@
 
 describe('Book Store app', () => {
   before(() => {
+    cy.visit('/login');
+  });
+  // const fName = 'testName';
+  // const lName = 'testSurname';
+
+  const uName = 'testUsername';
+  const uPassword = 'testPassword1!';
+  it('correct login user', () => {
+    cy.get('#userName')
+      .type(uName);
+
+    cy.get('#password')
+      .type(uPassword);
+
+    cy.get('#login')
+      .click();
+
+    cy.url()
+      .should('include', '/profile');
+
+    cy.get('#userName-value')
+      .should('contain', uName);
+  });
+
+  it.only('adding the book to shopping list', () => {
```

Comment:
Do not leave `.only()` in your code while merging. It is for debugging purposes only.

Reply:
fixed